picard/ui/ui_infodialog.py

```diff
@@ -1,132 +1,73 @@
 # -*- coding: utf-8 -*-
+
 # Automatically generated - don't edit.
 # Use `python setup.py build_ui` to update it.
 
-from PyQt4 import QtCore, QtGui
-
-try:
-    _fromUtf8 = QtCore.QString.fromUtf8
-except AttributeError:
-    _fromUtf8 = lambda s: s
-
-class ArtworkTable(QtGui.QTableWidget):
```

Comment:
Huh, this shouldn't be removed. Are you sure you rebased things against latest master?